src/main/java/meteordevelopment/meteorclient/systems/modules/world/VeinMiner.java

```diff
@@ -57,14 +57,14 @@ public class VeinMiner extends Module {
     private final Setting<FilterMode> blocksFilter = sgGeneral.add(new EnumSetting.Builder<FilterMode>()
         .name("blocks-filter")
         .description("Selection mode.")
-        .defaultValue(FilterMode.Whitelist)
+        .defaultValue(FilterMode.Blacklist)
         .build()
     );
 
     private final Setting<List<Block>> selectedBlocks = sgGeneral.add(new BlockListSetting.Builder()
         .name("blocks")
         .description("Which blocks to select.")
-        .defaultValue(Blocks.STONE, Blocks.DIRT, Blocks.GRASS)
```

Comment:
i meant to change Blocks.GRASS into Blocks.GRASS_BLOCK, not to replace all of them lol